Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -157,5 +157,32 @@ public async Task<List<FleetAttributes>> DescribeFleetAttributes(ComputeType com
                 return null;
             }
         }
+        
+        public async Task<DescribeFleetLocationResponse> DescribeFleetLocationAttributes(string fleetId)
```

Comment:
```suggestion
        public async Task<DescribeFleetLocationResponse> FindFirstFleetLocation(string fleetId)
```
Consumers of this method only get the first location, they don't get a full DescribeFleetLocationAttributes response, so the method naming is misleading.

Reply:
Changed